.github/workflows/provider-updoc.yml

```diff
@@ -2,13 +2,19 @@ name: Updoc
 
 on:
   workflow_call:
+    inputs:
+      publish-pack:
```

Comment:
nit: We may use a more explanatory name for this parameter:
```suggestion
      providers:
```

Reply:
Done.